features/step_definitions/cli.rb

```diff
@@ -198,9 +198,15 @@
   @result = @bg_processes.last.result
 end
 
-Given /^I check status of last background process$/ do
+Given /^I check status of( and pop)? last background process$/ do |pop|
   @bg_processes.last.wait(10)
   @result = @bg_processes.last.result
+  if pop
+    raise "0 background process left" unless @bg_processes.length > 0
```

Comment:
`@bg_processes.length` should >= 0, seems the condition always is unable to meet. Replace with `if  @bg_processes.length == 0`, the raise will be executed.

Reply:
@wangke19 `unless @bg_processes.length > 0` is equivalent to `if @bg_processes.length == 0`, so I prefer to not change it.